tests/common/snappi_tests/snappi_helpers.py

```diff
@@ -182,17 +182,29 @@ def get_ports(self, peer_device=None):
         retval = []
         for port in self.current_snappi_port_list:
             info_list = port.split('/')
-            dict_element = {
```

Comment:
as @amitpawar12 suggested, please add some comment to show example for breakout definition, and sample definition that can be used for _link.csv.

Reply:
@selldinesh can you update the comment to have an example for breakout?

Reply:
done